samples/in_process.go

```diff
@@ -95,6 +95,10 @@ func (t *SampleTest) initialize(
 		return fmt.Errorf("TempDir: %v", err)
 	}
 
+	// Disable writeback caching so that pid is always available in OpContext
+	t.MountConfig.DisableWritebackCaching = true
+	config.DisableWritebackCaching = true
```

Comment:
I find this way of implementation a bit funny - without line 99, `DisableWritebackCaching` for `sampleTest` won't be set as true, not sure if that matters? 
Maybe we can get rid of line 99 if all that matters is the config we pass into `fuse.Mount()`.

Reply:
We might get some pushback from repo owners because this does not seem to be the "sanest" way to do this.
But it is okay. We might get some suggestions on how best to do it. 

Misc note: DisableWritebackCaching is being disabled in a few other tests. We will no longer need that after this change. https://github.com/jacobsa/fuse/search?q=DisableWritebackCaching&type=Code

Reply:
Thanks for this! I didn't understand the testing framework well, fixed.